packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -23,18 +23,26 @@ export interface MeetingCallControlBarProps {
   onPeopleButtonClicked: () => void;
   mobileView: boolean;
   disableButtonsForLobbyPage: boolean;
+  meetingCallControlOptions?: MeetingCallControlOptions;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
-  // Disable the default participants button as meetings composite has its own participants button.
+  // Set the desired control buttons from the meetings composite. particiapantsButton is always false since there is the peopleButton.
   let callControlsOptions: CallControlOptions | false = {
+    ...props.meetingCallControlOptions,
     participantsButton: false,
-    screenShareButton: props.mobileView ? false : { disabled: props.disableButtonsForLobbyPage }
+    screenShareButton:
+      props.mobileView || !props.meetingCallControlOptions?.screenShareButton
+        ? false
+        : { disabled: props.disableButtonsForLobbyPage }
   };
 
+  // Helper function to determine if a meeting control bar button is enabled or not.
+  const isEnabled = (option: boolean | undefined): boolean => !(option === false);
```

Comment:
nitty nit: I'd make this a file-level private function. Why create a new function object on each render?